src/Discord.Net.WebSocket/Entities/Channels/SocketGuildChannel.cs

```diff
@@ -168,8 +184,26 @@ public Task DeleteAsync(RequestOptions options = null)
             }
         }
 
+        /// <summary>
+        ///     Gets the invites for this channel.
+        /// </summary>
+        /// <param name="options">The options to be used when sending the request.</param>
+        /// <returns>
+        ///     An awaitable <see cref="Task"/> containing a <see cref="IReadOnlyCollection{RestInviteMetaData}"/>.
```

Comment:
That doesn't tell people anything about what the content actually is. You can tell me it's a collection of `RestInviteMetadata`, but what the heck *is* this and how can it be used